snapcraft/commands/ovs-ssl-wrapper

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+export OVS_RUNDIR="${SNAP_COMMON}/run/switch/"
+
+# Optionally load SSL certificates if they are readable
+if [ -r "$SNAP_COMMON/data" ]; then
```

Comment:
In an aged install, the `$SNAP_COMMON/data` directory is readable while `${OVN_PKI_DIR}` is not. So in my example we would still see this error.

Would it be more appropriate to check whether the files within `${OVN_PKI_DIR}` are accessible?

Looking at `${SNAP}/ovn.env` it sources `"${SNAP_COMMON}/data/env/ovn.env"` which is probably what lead you to do the check on `"$SNAP_COMMON/data"` but with the inconsistency we have between fresh and older deployments I don't think we can.

Sourcing `/snap/microovn/current/ovn.env` does populate `OVN_PKI_DIR` even if it will complain about not being able to read `"${SNAP_COMMON}/data/env/ovn.env"`, so we could do that and deal with permissions for the data dir and/or permission check for `/snap/microovn/current/ovn.env` separately, whichever we find more appropriate.

Reply:
> Looking at ${SNAP}/ovn.env it sources "${SNAP_COMMON}/data/env/ovn.env" which is probably what lead you to do the check on "$SNAP_COMMON/data"

Exactly.

> Sourcing /snap/microovn/current/ovn.env does populate OVN_PKI_DIR even if it will complain about not being able to read

We could also conditionally source the `$SNAP_COMMON/data/env/ovn.env` in `$SNAP/ovn.env`. No sense in sourcing something that can't be read. We could then properly gate the `ovs-ssl-wrapper` based on accesibility to `$OVN_PKI_DIR`